src/udev/udev-manager.c

```diff
@@ -1166,7 +1166,7 @@ static int manager_listen_fds(Manager *manager) {
 
         int n = sd_listen_fds_with_names(/* unset_environment = */ true, &names);
         if (n < 0)
-                return n;
+                return log_error_errno(n, "Failed to listen on fds: %m");
```

Comment:
Hmm, we should merge this, but I think we want to provide some more feedback here. If there's something wrong the the fd setup, this is not enough info.

> systemd-udevd[721470]: Failed to listen on fds: Invalid argument

Reply:
Yeah, but the caller does not know why it fails.
Maybe be better to add debugging logs in the function ?